docs/pages/connect-your-client/introduction.mdx

```diff
@@ -0,0 +1,330 @@
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            example.com
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+</ScopedBlock>
+
+<ScopedBlock scope={["enterprise"]}>
+<Tabs>
+<TabItem label="Local user">
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            example.com
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+</TabItem>
+<TabItem label="SAML / OIDC user">
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice@example.com --auth=your-idp-connector
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            example.com
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+</TabItem>
+</Tabs>
+
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            mytennant.teleport.sh
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+Depending on how Teleport was configured for your network, you may need
+additional flags, such as `--auth=a_saml_connector`. Your administrator should
+provide the details required for your particular use case.
+
+</ScopedBlock>
+
+### Teleport Connect
+
+The Teleport Connect app provides all the same access to resources as `tsh` in
+a friendly graphic user interface. After [downloading](https://goteleport.com/download/)
+and installing Teleport Connect, you can log in and initiate sessions for
+Server and Database Access within a single window.
+
+1. Click **CONNECT** to connect to the Teleport cluster:
```

Comment:
I think we can remove the numbers from this list, since these list items also work as regular paragraphs.

Reply:
Sure, they work either way. I'm partial to numbered steps since they better indicate order, and IMHO make it easier for a reader switching focus between a doc and the actual product to remember where they were on the doc.

Do you feel strongly about removing them?

Reply:
For me, it's an aesthetic preference and makes these paragraphs more consistent with those in the rest of the page.  If you have reasons for keeping numbers here, that's okay for me!